coalib/parsing/DefaultArgParser.py

```diff
@@ -72,6 +72,12 @@ def default_arg_parser(formatter_class=None):
                             help='Files that will be analyzed will be '
                                  'restricted to those in the globs listed '
                                  'in this argument as well the files setting')
+    arg_parser.add_argument('--caching',
```

Comment:
subcommands are like `git commit` has an own `commit` command, we have no such things, we just want to group arguments. I think we should rather open an issue for that, can you do that?